ocaml/xapi/xapi_host.ml

```diff
@@ -442,7 +442,7 @@ let compute_evacuation_plan_wlb ~__context ~self =
       let target_uuid = List.hd (List.tl detail) in
       let target_host = Db.Host.get_by_uuid ~__context ~uuid:target_uuid in
       if
-        Db.Host.get_control_domain ~__context ~self:target_host != v
+        Db.Host.get_control_domain ~__context ~self:target_host = v
```

Comment:
oops

Reply:
Please double-check the change is correct.

> Note: if the vm being moved is dom0 then this is a power management rec and this check does not apply

I understood the comment as dom0 being excluded from the same-host check. Is this reasoning wrong?